src/pca_report_library/customer/graphs.py

```diff
@@ -38,88 +52,90 @@ def graph_builder(assessment_ID, labels):
+            json.dump(report_data, fp, indent=4)
 
     return success
 
 
-def time_ticks(x, pos):
-    return str(timedelta(seconds=int(x))).split(".")[0]
+def time_ticks(in_val, _):
+    """Time tick interval logic to return string."""
+    return str(timedelta(seconds=int(in_val))).split(".")[0]
 
 
 # Unique User Click Rate vs. Report Rate per Level of Deception
-def unique_user_click_rate_vs_report_rate_per_level_deception(reportData):
+def unique_user_click_rate_vs_report_rate_per_level_deception(report_data):
+    """Handle unique user click rate vs report rate logic."""
     # data to plot
     n_groups = 6
     click_rate = list()
     report_rate = list()
     ratio = list()
-    userReportProvided = reportData["User_Report_Provided"]
+    user_report_provided = report_data["User_Report_Provided"]
 
     # Pulls the Click Rate from the Report Data
-    for numlevel, value1 in reportData["Level"].items():
-        click_rate.append(reportData["Level"][numlevel]["Click_Rate"])
+    for numlevel, value1 in report_data["Level"].items():
+        click_rate.append(report_data["Level"][numlevel]["Click_Rate"])
 
-    if userReportProvided:
+    if user_report_provided:
 
         # Calculates the Report Rate from the Report Data
-        for numlevel, value1 in reportData["Level"].items():
+        for numlevel, value1 in report_data["Level"].items():
             report_rate.append(
                 round(
-                    reportData["Level"][numlevel]["User_Reports"]
-                    / reportData["Level"][numlevel]["Emails_Sent_Attempted"]
+                    report_data["Level"][numlevel]["User_Reports"]
+                    / report_data["Level"][numlevel]["Emails_Sent_Attempted"]
                     * 100,
                     2,
                 )
             )
 
         # Builds the report vs click ratio
-        for numlevel, value1 in reportData["Level"].items():
-            if reportData["Level"][numlevel]["User_Clicks"] == 0:
-                ratio.append(reportData["Level"][numlevel]["User_Reports"])
+        for numlevel, value1 in report_data["Level"].items():
+            if report_data["Level"][numlevel]["User_Clicks"] == 0:
+                ratio.append(report_data["Level"][numlevel]["User_Reports"])
             else:
                 ratio.append(
                     round(
-                        reportData["Level"][numlevel]["User_Reports"]
-                        / reportData["Level"][numlevel]["User_Clicks"],
+                        report_data["Level"][numlevel]["User_Reports"]
+                        / report_data["Level"][numlevel]["User_Clicks"],
                         1,
                     )
                 )
 
     else:
         # Sets the Report Rate to 0 when user reports are not provided.
-        for numlevel, value1 in reportData["Level"].items():
+        for numlevel, value1 in report_data["Level"].items():
             report_rate.append(0)
```

Comment:
I think this matches the intent.

```suggestion
        for _ in report_data["Level"].keys():
            report_rate.append(0)
```